### PR TITLE
Net5.0 update and upgrade from System.Data.SqlClient to Microsoft.Data.SqlClient

### PR DESCRIPTION
**Pull request by amodpandey**
# Update to Net5.0
# Removed FRAMEWORK code and dependencies
# migrated to use Microsoft.Data.SqlClient

### CODE REVIEW

**Comment by adamtoakley** [comment]
I would love for this to be added to the library!

File: Test/ElasticScale.Query.UnitTests/Microsoft.Azure.SqlDatabase.ElasticScale.Query.UnitTests.csproj [hunk 1]
```diff
@@ -1,15 +1,16 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>net452;netcoreapp2.0</TargetFrameworks>
+    <TargetFrameworks>net5.0</TargetFrameworks>
     <IsPackable>false</IsPackable>
+    <NoWarn>$(NoWarn);CS8073;</NoWarn>
```
Comment (caseycole589) [inline Test/ElasticScale.Query.UnitTests/Microsoft.Azure.SqlDatabase.ElasticScale.Query.UnitTests.csproj:5]: look up this code

**Comment by 8erh0f** [comment]
Please pull this merge request. We need an updated version that supports .Net6 and Microsoft.Data.SqlClient. Is there anybody out there with write access and time? It looks like this project is not going anywhere sinds 2019. Any alternatives?

**Comment by jeremy-holovacs-sp** [comment]
This may need to be forked.

**Comment by jeremy-holovacs-sp** [comment]
@scoriani my understanding is you were working on this issue, but we haven't heard anything. What is going on with this project?

File: Samples/EFCodeFirst/appsettings.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+  "ConnectionStrings": {
+    "DevelopmentDatabase": "Data Source=(localdb)\\v11.0;Initial Catalog=elasticscaledev;Integrated Security=True"
+  }
+}
```
Comment (stuartpa) [inline Samples/EFCodeFirst/appsettings.json:5]: Should this file be included in this PR?

File: Test/ElasticScale.ClientTestCommon/Microsoft.Azure.SqlDatabase.ElasticScale.ClientTestCommon.csproj [hunk 3]
```diff
@@ -1,13 +1,12 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>net451;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>net5.0</TargetFrameworks>
+    <NoWarn>$(NoWarn);SYSLIB0011;</NoWarn>
```
Comment (stuartpa) [inline Test/ElasticScale.ClientTestCommon/Microsoft.Azure.SqlDatabase.ElasticScale.ClientTestCommon.csproj:4]: Why are we suppressing this warning, should we have a GH related issue entered?

File: Samples/EFMultiTenant/appsettings.json [hunk 4]
```diff
@@ -0,0 +1,5 @@
+{
+  "ConnectionStrings": {
+    "DevelopmentDatabase": "Data Source=(localdb)\\v11.0;Initial Catalog=elasticscaledev;Integrated Security=True"
+  }
+}
```
Comment (stuartpa) [inline Samples/EFMultiTenant/appsettings.json:5]: Presume this file should not be included

**Pull request merged**